src/Database/Mysql/Driver/Mysql.php

```diff
@@ -54,15 +59,46 @@ public function update(Tenant $tenant): bool
             return false;
         }
 
-        return $this->process($tenant, [
-            'user' => "RENAME USER `{$config['oldUsername']}`@'{$config['host']}' TO `{$config['username']}`@'{$config['host']}'",
-        ]);
+        if (isset($config['allowedHost'])) {
+            $config['host'] = $config['allowedHost'];
+        }
+
+        $tempTenant = $tenant;
```

Comment:
Can we use a more meaningful variable than `$tempTenant`?

Reply:
$oldTenant good enough in that case?
We also use oldUsername in the config so would match those 2.

Reply:
Or originalTenant might actually work better?